### DISCUSSION TITLE
A slight increase in the amount of fitness value when the goal is to minimize

### DISCUSSION THREAD

**Discussion by zsbeheshti** [ahmedfgad/GeneticAlgorithmPython#186] [Q&A]
Dear all,

I use PyGAD for minimization (so I multiply the fitness function by the negative); Therefore, as the generations go forward, the fitness value of the best solution should decrease. But sometimes it increases by about 0.00000005 compared to the previous output. What is the reason?

Regards

**Comment by ahmedfgad** [comment]
PyGAD only works for maximization problems. Please formulate the fitness function to be a maximization function instead.
